### ISSUE TITLE
Node still showing after deletion

### ISSUE THREAD

**Issue by adrienpicard** [verbb/vizy#69]
**Description**
This might be an edge case, but when a node (here `blockquote`) is placed between two Vizy Blocks and then removed, it's not entirely deleted.

**Steps to reproduce**
https://cln.sh/EX1RGt

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.6](https://github.com/verbb/vizy/releases/tag/1.0.6)
